README.md

```diff
@@ -10,10 +12,30 @@
 
 4. Are there unexpected regions with higher stars (rare patterns)? Something non-trivial that may be discovered are "hidden gems" such as food cart street or local spots
 
+5. Are more active yelpers prone to giving higher ratings or more thorough reviews?
+
+6. Do some yelpers give generally negative/positive reviews?
+
+7. Are there common words present in reviews with similar ratings?
+
+8. How consistent are ratings with common keywords?
```

Comment:
We could probably explore these thoughts via question 2

Reply:
Agreed. If we do merge with (2), we can keep in mind the following (so far) to investigate:

**Q: What are frequent features of 1-star (negative) and 5-star (positive) text reviews for different categories of business? Are there nuanced patterns amongst major cities in the US?**

- Frequently used words
- Review length
- User's review count
- User's friend network
- User's average star rating

Maybe we can include these queries under question (2) or simply put them in the slides when we compile them this week?

Reply:
They do batch well together under this single topic.